airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/manifest_reference_resolver.py

```diff
@@ -7,16 +7,18 @@
 
 from airbyte_cdk.sources.declarative.parsers.custom_exceptions import CircularReferenceException, UndefinedReferenceException
 
+REF_TAG = "$ref"
```

Comment:
Why extract this outside of the class boundaries? Shouldn't this be private to `ManifestReferenceResolver`?

Reply:
I could go either way. It allowed me to clean things up a little - making both `is_ref` and `is_ref_key` `staticmethod`s, plus I didn't feel strongly that it should be owned by `ManifestReferenceResolver`. That said, it is only _used_ by `ManifestReferenceResolver`, so I can see an argument for keeping it private.